recipes/icu/all/conanfile.py

```diff
@@ -72,15 +79,18 @@ def build(self):
             tools.replace_in_file(run_configure_icu_file, "-MD", flags)
 
         self._workaround_icu_20545()
+        self._remove_scrptrun()
```

Comment:
what is this scrptrun?

Reply:
Before 65.1 this was something in the repository but it was not added to the build system, 65.1 adds it and the build breaks... according to the docs http://userguide.icu-project.org/layoutengine
> The ICU LayoutEngine is designed to process a run of text which is in a single font. It is written in a single direction (left-to-right or right-to-left), and is written in a single script. Clients can use ICU's Bidi processing to determine the direction of the text and use the ScriptRun class in icu/source/extra/scrptrun to find a run of text in the same script. Since the representation of font information is application specific, ICU cannot help clients find these runs of text.

I wouldn't be able to explain it to you 😓